fastlane_core/lib/fastlane_core/itunes_transporter.rb

```diff
@@ -415,6 +449,9 @@ def upload(app_id, dir)
       UI.verbose(@transporter_executor.build_upload_command(@user, 'YourPassword', actual_dir, @provider_short_name))
 
       begin
+        if Helper.mac?
+          Dir.chdir('/usr/local/itms')
```

Comment:
I think this might have messed with the tests.

Reply:
My bad. Test OK after remove chdir.